src/components/Checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styles from "./Checkbox.module.scss";
+import { color } from "@storybook/addon-knobs";
+
+export function Checkbox({value, name} = {}) {
+  return <input type="checkbox" name={name} value={value}/>
```

Comment:
I think, that a more common approach is to export something from css module, rather than apply it to the whole file in which it's imported. I'd suggest smth like

```scss
.checkbox {
// your styles
}
```

```js
import styles from "./Checkbox.module.scss";
// ...
return <input className={styles.checkbox} type="checkbox" name={name} value={value}/>
```

Additional question to think about - how we'll know when someone changed the checkbox value? Try to render it and play with it 😉

Reply:
I thought we want to change the styling to all checkboxes. But, yes your suggestion is a better practice, thanks:D

Reply:
Well, if we'll use `Checkbox` component for every checkbox component in our app... we'll apply the styles for every checkbox in the app + we'll scope the styles to the component only by leveraging the css modules